base_layer/core/src/proof_of_work/lwma_diff.rs

```diff
@@ -11,11 +11,8 @@ use crate::proof_of_work::{
     error::DifficultyAdjustmentError,
 };
 use std::{cmp, collections::VecDeque};
+use tari_transactions::consensus::{BLOCK_WINDOW, TARGET};
```

Comment:
It loses context though. Probably rename to DIFFICULTY_BLOCK_WINDOW and TARGET_BLOCK_INTERVAL